README.md

```diff
@@ -0,0 +1,128 @@
+
+![img](./banner.png)
+
+
+# Table of Contents
+
+1.  [Sovrin Token Plugins](#org581a5d8)
+    1.  [Install](#org6c5e3ee)
+    2.  [Development Environment](#org003878b)
+        1.  [Why cant I use my own local machine as my enviroment?](#orga843a17)
+        2.  [Requirements](#orgf42e059)
+        3.  [Installing docker-sync](#orge005da4)
+        4.  [Starting Dev Env](#orgb33d5f6)
+        5.  [Alternative for running tests](#orga76b3c6)
+    3.  [How To Contribute](#orgd389b14)
+
+<a href="https://www.apache.org/licenses/LICENSE-2.0.txt" target="_blank">![Hex.pm](https://img.shields.io/hexpm/l/plug.svg?style=plastic)</a>
+<a href="https://badge.fury.io/gh/sovrin-foundation%2Ftoken-plugin">[![GitHub version](https://badge.fury.io/gh/sovrin-foundation%2Flibsovtoken.svg)](https://badge.fury.io/gh/sovrin-foundation%2Ftoken-plugin)</a>
+
+<a id="org581a5d8"></a>
+
+# Sovrin Token Plugins
+
+This repo contains the plugins to create the Sovrin payment ledger plugins: Token and Fees
+
+
+<a id="org6c5e3ee"></a>
+
+## Install
+
+run the following command: 
+
+    ./build_indy_pool
+
+
+<a id="org003878b"></a>
+
+## Development Environment
+
+
+<a id="orga843a17"></a>
+
+### Why cant I use my own local machine as my enviroment?
+
+Due to some compatibility issues we have decided that a docker enviroment would be the easiest way
```

Comment:
recommend changing this to:
You are welcome to do that. We provide a docker environment that comes pre-loaded with all required dependencies and consider this to be the easiest way for any new developer to get setup and start testing their code.